plugins/experimental/ssl_cert_loader/domain-tree.cc

```diff
@@ -173,10 +173,11 @@ DomainNameTree::insert(std::string key, void *payload, int order)
 
       new_node->parent = node->parent;
       new_node->children.push_back(node);
+      node->parent = new_node;
```

Comment:
I think this is correct, but I wonder why this hasn't been a problem. The code is just not used that much?